### PR TITLE
Update themes development instructions

### PR DESCRIPTION
**Pull request by laurieontech**
With migration to the new repo the official themes development docs are outdated. This updates them.

### CODE REVIEW

File: docs/contributing/code-contributions.md [hunk 1]
```diff
@@ -125,16 +125,12 @@ Check [Debugging the build process](/docs/debugging-the-build-process/) page to
...
-[set up your local dev environment](/contributing/setting-up-your-local-dev-environment/)
-and that you're on the latest version of `gatsby-dev-cli`.
-
-- In the Gatsby monorepo find the starter in the `/starters` directory that uses the theme you want to work on
-- Navigate to that directory, e.g. `cd starters/gatsby-starter-blog-theme`
-- Install dependencies: `yarn`
-- Run Gatsby Dev CLI to sync theme files, referencing the appropriate theme: `gatsby-dev --packages gatsby-theme-blog`
-- In another tab run the starter: `yarn develop`
-- Edit the theme files, you'll see changes automatically copied over and update in your starter.
+Themes live in their own [repo](https://github.com/gatsbyjs/themes). The themes themselves live in the `packages` directory and starters using them are in the `starters` directory. In order to work on a theme find the starter that uses it and do the following:
```
Comment (AishaBlake) [inline docs/contributing/code-contributions.md:128]:
```suggestion
Themes live in their own [repo](https://github.com/gatsbyjs/themes). The themes themselves live in the `packages` directory and starters using them are in the `starters` directory. In order to work on a theme, find the starter that uses it and do the following:
```

You and these commas. 😂 💖

File: docs/contributing/code-contributions.md [hunk 2]
```diff
@@ -125,16 +125,12 @@ Check [Debugging the build process](/docs/debugging-the-build-process/) page to
...
+
+1. Run `yarn` in your terminal in the root of the repo.
+2. Still in the repo root run `yarn workspace <starter name> develop`.
```
Comment (AishaBlake) [inline docs/contributing/code-contributions.md:131]:
```suggestion
2. Still in the repo root, run `yarn workspace <starter name> develop`.
```

File: docs/contributing/code-contributions.md [hunk 3]
```diff
@@ -125,16 +125,12 @@ Check [Debugging the build process](/docs/debugging-the-build-process/) page to
...
+
+From there you can make changes in the theme and seem them reflected in the running site.
```
Comment (AishaBlake) [inline docs/contributing/code-contributions.md:133]:
```suggestion
From there, you can make changes in the theme and see them reflected in the running site.
```

File: docs/contributing/code-contributions.md [hunk 4]
```diff
@@ -125,16 +125,12 @@ Check [Debugging the build process](/docs/debugging-the-build-process/) page to
 This section is for official theme development in Gatsby's monorepo. If you are looking
```
Comment (AishaBlake) [inline docs/contributing/code-contributions.md:125]: This little intro should probably be changed as well!

**Comment by laurieontech** [comment]
Me, and, my, commas 😂 Intro is updated now!

**Pull request merged**